matrix_client/room.py

```diff
@@ -493,6 +493,80 @@ def backfill_previous_messages(self, reverse=False, limit=10):
         for event in events:
             self._put_event(event)
 
+    def modify_user_power_levels(self, users=None, users_default=None):
+        """Modify the power level for a subset of users
+        Args:
+            users(dict): Power levels to assign to specific users, in the form
+                {"@name0:host0": 10, "@name1:host1": 100, "@name3:host3", None}
+                A level of None causes the user to revert to the default level
+                as specified by users_default.
+            users_default(int): Default power level for users in the room
+        Returns:
+            True if successful, False if not
+        """
+        try:
+            content = self.client.api.get_power_levels(self.room_id)
+            if users_default:
+                content["users_default"] = users_default
+
+            if users:
+                if "users" in content:
+                    content["users"].update(users)
+                else:
+                    content["users"] = users
+
+                # Remove any keys with value None
+                for user in list(content["users"].keys()):
```

Comment:
Not familiar enough with differences between python2/3. Is there some
python2-specific reason that you can't iterate directly through a keys
object rather than converting it to a list first?